Makefile

```diff
@@ -49,6 +49,8 @@ Makefile.common.$(MAKE_BRANCH):
 include Makefile.common
 
 ifeq ($(ARCH),arm64)
+# Forces ARM64 build image to be used in a crosscompilation run.
+CALICO_BUILD:=$(CALICO_BUILD)-$(ARCH)
```

Comment:
When docker pulls `go-build:v0.55` for `arm64` it tags it incorrectly and use it for all other architectures which causes problems.
In this line we make sure that image pulled by docker for `arm64` process is tagged correctly in local repo. `go-build:v0.55-arm64`